do.py

```diff
@@ -831,6 +855,11 @@ def verbose(args):
     main(args)
     _verbosity = False
 
+def quiet(args):
```

Comment:
As a general rule, any function should have a docstring. Here, as single line would suffice, but it should be there.